db_stress_tool/no_batched_ops_stress.cc

```diff
@@ -1547,6 +1552,10 @@ class NonBatchedOpsStressTest : public StressTest {
     } else if (injected_error_count > 0 && IsRetryableInjectedError(s)) {
```

Comment:
I think this will miss the case when all injected errors should be ignored?

Reply:
Yeah we don't verify if an ignore injected error is indeed returned to the users. That's because the "ignore" part is crash test functionality, not really indicating our code base problem. The other side of the story "not returning the injected error when there is one and not ignored" might indicate the code base problem of ignoring or mishandling error.